moveit_planners/chomp/chomp_interface/include/chomp_interface/chomp_interface.h

```diff
@@ -34,35 +34,35 @@
 
 /* Author: E. Gil Jones */
 
-#ifndef _CHOMP_INTERFACE_ROS_H_
-#define _CHOMP_INTERFACE_ROS_H_
+#ifndef CHOMP_INTERFACE_CHOMP_INTERFACE_H
+#define CHOMP_INTERFACE_CHOMP_INTERFACE_H
 
 #include <chomp_motion_planner/chomp_planner.h>
 #include <chomp_motion_planner/chomp_parameters.h>
 #include <ros/ros.h>
 
-namespace chomp_interface_ros
+namespace chomp_interface
 {
-/** @class CHOMPInterfaceROS */
-class CHOMPInterfaceROS : public chomp::ChompPlanner
+class CHOMPInterface : public chomp::ChompPlanner
 {
 public:
-  CHOMPInterfaceROS(const planning_models::RobotModelConstPtr &kmodel);
+  CHOMPInterface(const moveit::core::RobotModelConstPtr& kmodel);
 
-  const chomp::ChompParameters& getParams() const {
+  const chomp::ChompParameters& getParams() const
+  {
     return params_;
   }
 
 protected:
-
   /** @brief Configure everything using the param server */
   void loadParams();
 
-  ros::NodeHandle nh_; /// The ROS node handle
+  ros::NodeHandle nh_;  /// The ROS node handle
 
   chomp::ChompParameters params_;
 };
 
+typedef boost::shared_ptr<CHOMPInterface> CHOMPInterfacePtr;
```

Comment:
Perhaps it should use the MOVEIT_CLASS_FORWARD macro to create all the boost pointer types